components/edit-collective/sections/HostVirtualCards.js

```diff
@@ -358,7 +370,7 @@ const HostVirtualCards = props => {
           </Box>
         </AddCardPlaceholder>
         {data.host.hostedVirtualCards.nodes.map(vc => (
-          <VirtualCard key={vc.id} {...vc} onUpdate={refetch} hasActions />
+          <VirtualCard key={vc.id} {...vc} onUpdate={refetch} editHandler={handleEdit(vc)} hasActions />
```

Comment:
@kewitz : I was wondering if the function `handleEdit` is really necessary as it's only used here; 

```suggestion
          <VirtualCard key={vc.id} {...vc} onUpdate={refetch} editHandler={setEditingVirtualCard(vc)} hasActions />
```